### PR TITLE
packr: update test to support go 1.16

### PR DESCRIPTION
**Pull request by roopakv**
This should fix packr for go1.16

### CODE REVIEW

File: Formula/packr.rb [hunk 1]
```diff
@@ -56,6 +56,7 @@ def install
 
     system "go", "mod", "init", "example"
     system bin/"packr"
+    system "go" "get" "github.com/gobuffalo/packr"
```
Comment (carlocab) [inline Formula/packr.rb:59]:
```suggestion
    system "go", "get", "github.com/gobuffalo/packr"
```

**Comment by carlocab** [comment]
Whoops; `audit` does not like `go get`:

```
==> FAILED
Formula/packr.rb:59:5: C: Do not use go get. Please ask upstream to implement Go vendoring
    system "go", "get", "github.com/gobuffalo/packr"
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

**Comment by roopakv** [comment]
@carlocab trying something else.

the thing is i have a feeling this is not going to work with go1.15... lets see :)

File: Formula/packr.rb [hunk 2]
```diff
@@ -56,6 +56,11 @@ def install
 
     system "go", "mod", "init", "example"
     system bin/"packr"
+    # Bypass `go get` check. Needed for go 1.16
+    touch "goget_packr.sh"
+    chmod 0755, "goget_packr.sh"
+    inreplace "goget_packr.sh", "", "go get github.com/gobuffalo/packr"
+    system "./goget_packr.sh"
```
Comment (SMillerDev) [inline Formula/packr.rb:63]: could this be `go mod download` instead?

Comment (roopakv) [reply Formula/packr.rb:63]: Maybe? I'm just copying what the go 1.16 changes said to do.

https://blog.golang.org/go116-module-changes

I'm not sure if go mod download fixes things with 1.16. let me know what you think we should do

Comment (SMillerDev) [reply Formula/packr.rb:63]: I'd rather try and avoid tricking the audit. Possible solutions in my book:
- Figure out how to make Go create this in a mod file
- Manually add a modfile here

Comment (Bo98) [reply Formula/packr.rb:63]: I had a look at this. Looks like the formula is also missing the newer `packr2` binary. This is what I did to get it working: #71748.

**Comment by carlocab** [comment]
Superseded by #71748.

**Pull request closed without merging**